docs/src/examples/components/Table/BestPractices/TableBestPractices.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react'
+
+import { Text } from '@fluentui/react'
+import ComponentBestPractices from 'docs/src/components/ComponentBestPractices'
+import { link } from '../../../../utils/helpers'
+
+const doList = [
+  <Text>
+    Choose desired accessibility behavior for grid cell depending on the use case.(Check the{' '}
+    {link('Behaviors', '/components/table/accessibility')} section).
+  </Text>,
+  'Provide label to the Table component using `aria-label` or `aria-labelledby` prop.',
+  'Provide label to the Row component using `aria-label` or `aria-labelledby` prop. If not, then each cell of the row is narrated by screen reader.',
+  'Provide label to the table header column, if cell has no content.',
+  'Stop event propagation, when you will add actionable element into the grid cell.',
```

Comment:
If you use actionable element or component in the cell, stop propagation of the click event if you do not want the interacion with the element to perform row click action.